packages/bundle-size/README.md

```diff
@@ -0,0 +1,39 @@
+## `@fluentui/bundle-size`
+
+A tool to measure bundle size locally and on CI.
+
+**To ensure that we are measuring the same artifacts as customers `bundle-size` requires to build packages first.**
+
```

Comment:
**suggestion:** I'd recommend to follow "standard" chapters for a package documentation like following:

```
### Install
- not applicable atm

### Usage

- I'd put fixtures and measures example in here

### API

- not applicable atm

### Commands

#### measure
```

wdyt?

Reply:
Makes sense 👍 Applied in 25b135e.